lib/active_force/sobject.rb

```diff
@@ -129,6 +129,14 @@ def self.create! args
       new(args).create!
     end
 
+    def self.update(id, attributes)
+      new(attributes.merge(id: id)).update
```

Comment:
How would the `nil` fields be handled? I was digging through the source the other day, and I noticed this piece of code:
https://github.com/Beyond-Finance/active_force/blob/675aad3f19be1ce0a52e3483b93b01a4032776af/lib/active_force/sobject.rb#L247

~It's used inside of `update`. With this record being new, wouldn't all of the attributes be marked as dirty?~ Nevermind, I think that this is fine.